### PR TITLE
AZ bug: Dependents over 17 not being counted in AZ-140 10b

### PR DESCRIPTION
**Pull request by rickreyhsig**
https://www.pivotaltracker.com/story/show/186716200

### XML fix
Screen Shot 2023-12-28 at 1 31 26 PM

### PDF fix
Screen Shot 2023-12-28 at 1 31 34 PM

### CODE REVIEW

File: app/models/state_file_az_intake.rb [hunk 1]
```diff
@@ -111,11 +111,11 @@ def federal_dependent_count_under_17
   end
 
   def federal_dependent_count_over_17
-    self.dependents.select{ |dependent| dependent.age >= 17 }.length
+    self.dependents.select{ |dependent| dependent.age >= 17 && !dependent.ask_senior_questions? }.length
   end
 
   def qualifying_parents_and_grandparents
-    dependents.select(&:ask_senior_questions?).length
+    self.dependents.select(&:ask_senior_questions?).length
```
Comment (rickreyhsig) [inline app/models/state_file_az_intake.rb:118]: Adding the self to keep consistent with other two

File: app/models/state_file_az_intake.rb [hunk 2]
```diff
@@ -111,11 +111,11 @@ def federal_dependent_count_under_17
   end
 
   def federal_dependent_count_over_17
```
Comment (tofarr) [inline app/models/state_file_az_intake.rb:113]: I wonder should we rename this method / create a new, given that it now described a situation where the dependent is over 17 but not a senior?

Comment (rickreyhsig) [reply app/models/state_file_az_intake.rb:113]: Yep, good catch. Will rename!

Comment (rickreyhsig) [reply app/models/state_file_az_intake.rb]: Done

**Review by tofarr — APPROVED** [review]

**Pull request merged**